Packs/ML/Scripts/HashIncidentsFields/HashIncidentsFields.py

```diff
@@ -0,0 +1,75 @@
+import demistomock as demisto
+from CommonServerPython import *
+# from CommonServerUserPython import *
+import hashlib
+import pickle
+import uuid
+
+
+def hash_multiple(value):
+    if isinstance(value, list):
+        if not value:
+            return []
+        else:
+            return [hash_multiple(value[0])] + hash_multiple(value[1:])
+    if isinstance(value, dict):
+        for k, v in value.items():
+            value[k] = hash_multiple(v)
+        return value
+    else:
+        return hashlib.md5(str(value).encode('utf-8')).hexdigest()
+
+
+def get_context(incident_id):
+    res = demisto.executeCommand("getContext", {'id': incident_id})
+    try:
+        return res[0]['Contents'].get('context') or {}
+    except Exception:
+        return {}
+
+
+def hash_incident():
+    args = demisto.args()
+    fieldsToHash_list = args.pop('fieldsToHash', '').split(',')
+    fieldsToHash_list = set([x.strip() for x in fieldsToHash_list if x])
+    res = demisto.executeCommand('GetIncidentsByQuery', args)
+    if is_error(res):
+        return_error(get_error(res))
+    else:
+        entry = res[0]
+        incident_list = json.loads(entry['Contents'])
+        new_incident_list = []
+        for incident in incident_list:
+            for field in fieldsToHash_list:
+                if field in incident:
+                    incident[field] = hash_multiple(incident.get(field))
+            new_ctx = {}
+            if demisto.args().get('contextKeys', '').split(","):
```

Comment:
This condition is not good, the expression always contains empty string.
you need first to extract the context keys like that:
`context_keys = [x for x in demisto.args().get('contextKeys', '').split(",") if x]`
and than you can use `if context_keys: ...`